_posts/2017-08-16-node_starter_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Starters Guide to Node.js"
+date: 2017-08-16 00:00:00 +530
+author: Hritvik Taneja
+category:
+tags:
+- node
+- Node.js
+- npm
+---
+## `console.log('Hello World')`
+
+I assume that most of you are here to learn about Node.js. So, lets get started!!
+
+## What is Node.js ?
+
+**If you go by docs:** [Node.js®](https://nodejs.org/en/) is a JavaScript runtime built on [Chrome's V8 JavaScript engine](https://developers.google.com/v8/). Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, [npm](https://www.npmjs.com/), is the largest ecosystem of open source libraries in the world.
```

Comment:
Since we're saying all these words which sound a bit complex, it is worthwhile to explore them on a basic level at least. A lot of people reading this are new to all this in general, and "event-driven" or "non-blocking" might not make a lot of sense to them